src/main/java/com/google/sps/data/Project.java

```diff
@@ -46,6 +53,11 @@ public ProjectBuilder withSummary(String summary) {
       return this;
     }
 
+    public ProjectBuilder withUrl(String url) {
+      this.url = url;
+      return this;
+    }
+
```

Comment:
Did you implement "with" builder functions for title or summary?

Reply:
Yeah, the builder pattern had already been implemented, I only added this new method.